frontend/src/app/components/send-coins-form/send-coins-form.component.html

```diff
@@ -1,29 +1,26 @@
 <div id="container" [ngClass]="theme">
-  
-  <loader-component *ngIf="pageLoading" [theme]="loaderTheme"></loader-component>
-  
```

Comment:
why did we delete this?

Reply:
You did actually read the whole thing ;)

Because we already have loading in the dropdown, so we dont need a second loading which makes the page feel much slower and have a slower interaction time.

Its a UX change.